send/scim_atlassian_mu.pm

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
The name of this script is not proper. It should be "scim_atlassian_mu". The file extension "pm" means Perl module.